### ISSUE TITLE
Distribution of more flavours of wheels

### ISSUE THREAD

**Issue by FlorianWilhelm** [blue-yonder/turbodbc#89]
It would be really nice to have also wheels for Linux (`manylinux1_x86_64`) and (`cp34m`) for Turbodbc on PyPI. Many Cloudera and in general Linux distributions only provide Python 3.4. Thanks a bunch for your great work!

**Comment by MathMagique** [comment]
Hi Florian! You are absolutely right, those wheels would make installing turbodbc that much easier.

**Comment by xhochy** [comment]
@MathMagique We would then build wheels that have `unixodbc` bundled? Then I could build a quick Dockerfile that will emit these wheels and also make macOS wheels via a separate script.

@FlorianWilhelm For the time being, I've updated the `conda` packages to the latest version: https://github.com/conda-forge/turbodbc-feedstock / https://anaconda.org/conda-forge/turbodbc

**Comment by MathMagique** [comment]
@xhochy Yes, I think that would be the most reliable thing to do. Many Linux variants use unixodbc 2.3.x in the meantime (Ubuntu 16+, Debian 8+), so I would suggest to bundle the latest officially released version (2.3.4) with the wheels.

Probably it makes sense to add some build information to turbodbc so users can query the unixodbc version used during the build.

**Comment by MathMagique** [comment]
Hi nrhy! Sadly, the conversation did not continue with much momentum. In the meantime, you can use the `conda` package manager to install the precompiled [turbodbc conda packages](https://anaconda.org/conda-forge/turbodbc).

**Comment by nrhy** [comment]
Unfortunately conda is not supported in my workspace :(

**Comment by MathMagique** [comment]
That's unfortunate. In this case, I guess that until official manylinux wheels can be provided, you would need to create wheels for your specific operating system (`pip wheel` should help). Just make sure you build the wheel in an environment with NumPy/Arrow installed so these add-ons are built as well.

You can then distribute the wheel among your colleagues, e.g., via [devpi](https://devpi.net/docs/devpi/devpi/stable/%2Bd/index.html) or file transfer. If they share the same OS, you should be safe, although not very convenient.

**Comment by nrhy** [comment]
@MathMagique Yeah, it's what I've been doing. It'll have to do for now.

Thank you!

**Comment by mpeleshenko** [comment]
+1 on this. Having official Windows + Linux wheels available on pypi.org for at least Python 3.5+ would make installing turbodbc much easier. I remember when pyarrow starting putting out wheels on pypi in addition to their conda-forge builds.

**Comment by xhochy** [comment]
@mpeleshenko Can you help with implementing this? For the Linux wheels, a base would be to construct a docker image that contains all dependencies compiled for the manylinux1 image as in seen in `pyarrow`: https://github.com/apache/arrow/tree/master/python/manylinux1

**Comment by mpeleshenko** [comment]
@xhochy Sure. I’ll give it a shot. By the way, I saw you had previously opened https://github.com/blue-yonder/turbodbc/pull/108. Is that one no longer viable?

**Comment by xhochy** [comment]
@mpeleshenko That was a first shot, feel free to take this as a basline.

**Comment by yputter** [comment]
Azure Functions (Consumption Plan, Linux) also cannot install turbodbc through requirements.txt, probably missing the non-pythonic dependencies. I'm all new to this, but I came across conda-press and conda-smithy, maybe that could help with building and distributing "fat" wheels?
